homeassistant/components/switchbot/strings.json

```diff
@@ -58,5 +58,56 @@
         }
       }
     }
+  },
+  "entity": {
+    "binary_sensor": {
+      "calibration": {
+        "name": "Calibration"
+      },
+      "motion": {
+        "name": "[%key:component::binary_sensor::entity_component::motion::name%]"
+      },
+      "door_open": {
+        "name": "[%key:component::binary_sensor::entity_component::door::name%]"
+      },
+      "door_timeout": {
+        "name": "Door timeout"
+      },
+      "light": {
+        "name": "[%key:component::binary_sensor::entity_component::light::name%]"
+      },
+      "door_unclosed_alarm": {
+        "name": "Door unclosed alarm"
```

Comment:
Do we really need to prefix this all with "door"?

Reply:
Should be OK without "door" prefix, I've removed it